tests/CMakeLists.txt

```diff
@@ -202,8 +202,7 @@ if(WITH_PYTHON)
   add_fdb_test(TEST_FILES fast/RandomUnitTests.toml)
   add_fdb_test(TEST_FILES fast/ReadHotDetectionCorrectness.toml IGNORE) # TODO re-enable once read hot detection is enabled.
   add_fdb_test(TEST_FILES fast/ReportConflictingKeys.toml)
-  add_fdb_test(TEST_FILES fast/RESTKmsConnectorUnit.toml IGNORE)
-  add_fdb_test(TEST_FILES fast/RESTUtilsUnit.toml IGNORE)
+  add_fdb_test(TEST_FILES fast/RESTUnit.toml)
```

Comment:
should this still be ignored? if this is just a utility to run all of the rest tests, we probably either want to ignore it by default or given it a much lower test weight, since these will also be run by all of the regular unit test runners

Reply:
Thought process was combining multiple unit tests into single one might make the test run not _that_ fast anymore, however, I'm not aware means to specify lower weight, if possible, certainly worth exploring the option. 
Also, how about: I check in with lower weights and if still the tests is run way too many times, shall IGNORE it. Sounds reasonable?

Reply:
Or moving the file to `tests/rare` does the trick?

Reply:
the test directories are ignored in the new test harness

Reply:
yeah I had the same suspicion, checking the new mechanism to specify test weightage

Reply:
Based on offline discussion, I added IGNORE to the `RESTUnit.toml` Thanks